GA-worksheet.md

```diff
@@ -25,45 +25,51 @@
 
 ##Project #2 Part #2: Style the Header with CSS
 
-1. What advantages does the `<link>` tag have over the `<style>` tag?
+1. What advantages does the `<link>` tag have over the `<style>` tag? The <link> tag lets you include CSS from an external file. 
 
-2. What is normalize.css?
+2. What is normalize.css? It's a CSS file that preserve the CSS through any broswer. 
 
-3. Why do we need to add a padding property to the `<ul>` tag?
+3. Why do we need to add a padding property to the `<ul>` tag? We need to add a padding property to the <ul> tag because
+it centers the items on the list. 
 
-4. What does the “inline” property do to a list of items?
+4. What does the “inline” property do to a list of items? It makes every item listed to locate on the same line. 
 
 5. What do the following padding values do?
 
     padding: 25px 50px 75px 100px;
 
-    * top padding is __px
-    * right padding is __px
-    * bottom padding is __px
-    * left padding is __px
+    * top padding is 25 px
+    * right padding is 50 px
+    * bottom padding is 75 px
+    * left padding is 100 px
 
     padding: 20px;
 
-    * top padding is __px
-    * right padding is __px
-    * bottom padding is __px
-    * left padding is __px
+    * top padding is 20 px
```

Comment:
This should be 20px on all sides. When the padding property is given one value, the value applies to all sides of the 'box'.